metapool/sample_sheet.py

```diff
@@ -821,18 +947,11 @@ class MetagenomicSampleSheetv100(KLSampleSheet):
     # Note that there doesn't appear to be a difference between 95, 99, and 100
     # beyond the value observed in 'Well_description' column. The real
     # difference is between standard_metag and abs_quant_metag.
-
-    # Marks change from 'Metagenomics' to 'Metagenomic' - encapsulate this
-    # change: TODO.
-
-    # Note: Remove syndna_pool_number as that was part of the purpose of
-    # making this change. Also, it's always going to be empty or worse have
-    # a value that won't be checked.
     data_columns = ['Sample_ID', 'Sample_Name', 'Sample_Plate', 'well_id_384',
                     'I7_Index_ID', 'index', 'I5_Index_ID', 'index2',
                     'Sample_Project', 'Well_description']
 
-    # For now, assume only MetagenomicSampleSheetv100 (and v95, v99) contains
+    # For now, assume only MetagenomicSampleSheetv101, v100, v95, v99 contains
```

Comment:
To stop adding to this list, why not write it as: `For now, assume only AbsQuantSampleSheetv10 doesn't contain 'contains_replicates', the rest do. `